ui/apps/platform/src/Containers/Clusters/InitBundles/InitBundleWizardStep2.tsx

```diff
@@ -20,20 +20,19 @@ function InitBundleWizardStep2({ errorMessage, formik }: InitBundleWizardStep2Pr
     return (
         <Flex direction={{ default: 'column' }}>
             <Title headingLevel="h2">Download bundle</Title>
+            {installation === 'Operator' ? (
+                <SecureClusterUsingOperator headingLevel={headingLevel} />
+            ) : (
+                <SecureClusterUsingHelmChart headingLevel={headingLevel} />
+            )}
             <Alert
                 variant="info"
                 isInline
-                title="A cluster init bundle can only be downloaded once"
+                title="You can download the YAML file only once when you create a cluster init bundle"
```

Comment:
Should this have a period?

Reply:
I will add a period because it is a sentence.

Follow up post-MVP: PatternFly examples and guidelines suggest a concise title (which did not come to mind when I reviewed the draft wording) and then 2 sentences as content.